src/__experimental__/components/input-icon-toggle/input-icon-toggle.style.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import BaseTheme from 'style/themes/base';
+
+const InputIconToggleStyle = styled.label`
+  color: ${getColor};
+  background-color: ${getBackground};
+  border-left: 1px solid #668592;
+  cursor: pointer;
+  padding: 6px;
+  margin: -2px -6px; /*correction for parent element padding*/
+`;
+
+InputIconToggleStyle.defaultProps = {
+  theme: BaseTheme
+};
+
+function getColor({ theme, isHovered, disabled }) {
+  const { colors } = theme;
+
+  if (disabled) {
+    return colors.text.disabled;
+  }
+
+  if (isHovered) {
+    return colors.text.white;
+  }
+
+  return colors.text.body;
+}
+
+function getBackground({ theme, isHovered, disabled }) {
+  const { colors } = theme;
+
+  if (disabled) {
+    return colors.secondary;
```

Comment:
Are these the correct colours?

Reply:
these are the only colors available through current base theme config

Reply:
![image](https://user-images.githubusercontent.com/26925206/53731036-74019c80-3e71-11e9-91b5-2512aa840b5b.png)

Reply:
fixed